server/chat-plugins/scavengers.ts

```diff
@@ -27,30 +47,22 @@ const HOST_DATA_FILE = 'config/chat-plugins/scavhostdata.json';
 const PLAYER_DATA_FILE = 'config/chat-plugins/scavplayerdata.json';
 const DATABASE_FILE = 'config/chat-plugins/scavhunts.json';
 
-const SCAVENGE_REGEX = /^((?:\s)?(?:\/{2,}|[^\w/]+)|\s\/)?(?:\s)?(?:s\W?cavenge|s\W?cav(?:engers)? guess|d\W?t|d\W?ata|d\W?etails|g\W?(?:uess)?|v)\b/i; // a regex of some of all the possible slips for leaks.
+const SCAV_REGEX_STRING = "^((?:\\s)?(?:[^\\w]+)|\\s\\/)?(?:\\s)?" +
+	"(?:s\\W?cavenge|s\\W?cav(?:engers)? guess|d\\W?t|d\\W?ata|d\\W?etails|g\\W?(?:uess)?|v)\\b";
+const SCAVENGE_REGEX = new RegExp(SCAV_REGEX_STRING, 'i');
```

Comment:
Why this? The old regex should definitely already bypass the line length limit.

```suggestion
// a regex of some of all the possible slips for leaks.
const ACCIDENTAL_LEAKS = /^((?:\s)?(?:\/{2,}|[^\w/]+)|\s\/)?(?:\s)?(?:s\W?cavenge|s\W?cav(?:engers)? guess|d\W?t|d\W?ata|d\W?etails|g\W?(?:uess)?|v)\b/i;
```

Also I would probably rename this `ACCIDENTAL_LEAKS` to make it clearer what it means.